otter/utils.py

```diff
@@ -364,6 +380,31 @@ def _get_logger(cls):
         return cls._logger_instance
 
 
+class QueueLoggingHandler(logging.Handler):
+    """The logging handler that writes INFO messages to the log_queue.
+    This handler is set up when the result_queue is passed to
+    `otter grade`
+
+    Args:
+        logging (Hanlder): the base class for this handler
+    """
+    def __init__(self, log_queue: Queue):
+        """set the log level to `logging.INFO` when handler created
+
+        Args:
+            log_queue (multiprocessing.Queue): the queue this handler writes to
+        """
```

Comment:
`__init__` docstring is not needed. Move the `Args` here into the class docstring.